app/views/content/is-teaching-right-for-me/qualifications-you-need-to-teach.md

```diff
@@ -61,6 +61,12 @@ You can do a subject knowledge enhancement course before or sometimes during you
 
 [Find out more about subject knowledge enhancement courses](/how-to-apply-for-teacher-training/subject-knowledge-enhancement).
 
+## If you do not have a degree
+
```

Comment:
@MaxineCl Do we need to add a line here to say something like

**If you do not already have a degree, you can [train to be a teacher as part of your bachelor’s degree](/train-to-be-a-teacher/if-you-dont-have-a-degree) and get qualified teacher status.**

**Or** you may be eligible to apply for a new teacher degree apprenticeship.....


Otherwise we are only giving them half of the undergrad info?